extensions/non-common/README.md

```diff
@@ -0,0 +1,29 @@
+# Non-Common Properties Extension Specification (`-`)
```

Comment:
Rename?

Reply:
Yes, but there are still a lot more things to discuss, see #413.